Assets/Code/Basic Implementation/GoogleAdmob.cs

```diff
@@ -131,6 +131,7 @@ private void HandleRewardedAdClosed(object sender, EventArgs args)
             _rewardedAd.OnUserEarnedReward -= HandleUserEarnedReward;
             // Called when the ad is closed.
             _rewardedAd.OnAdClosed -= HandleRewardedAdClosed;
+            */
 
         }
```

Comment:
and google admob it being created without an abstraction. The correct way is implement an adapter pattern. It will allows us to change the ad mediator company easily